backend/seed.py

```
@@ -26,38 +27,131 @@
+                quiz_question_option = QuizQuestionOption(
+                    question_id=question.id,
+                    option_text=option["option_text"],
+                    is_correct=option["is_correct"],
+                    option_type=option["option_type"],
+                )
+                db.session.add(quiz_question_option)
+    db.session.commit()
+    logger.info("Quiz questions loaded successfully.")
+
+
+def load_hints():
```

Ruff (0.8.2)

Missing return type annotation for public function `load_hints`

Add return type annotation: `None`

(ANN201)